pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1217,6 +1217,14 @@ self: super: {
     })];
   });
 
+  # Remove unecessary constraint:
+  # https://github.com/agrafix/superbuffer/pull/2
+  superbuffer = overrideCabal super.superbuffer (drv: {
+    postPatch = ''
+      sed -i 's#QuickCheck < 2.10#QuickCheck < 2.13#' superbuffer.cabal
```

Comment:
If you wanted to go all out you could `fetchpatch` that change from https://github.com/agrafix/superbuffer/pull/2.patch and apply it instead of using `sed`. It's not a big difference, but the advantage is that the patch will fail to apply once an update comes out that fixes the issue and then we'll notice that the patch can be removed.